app/controllers/placements/support/schools/mentors_controller.rb

```diff
@@ -4,8 +4,46 @@ class Placements::Support::Schools::MentorsController < Placements::Support::App
 
   def show; end
 
+  def index; end
+
+  def new
+    @mentor_form = mentor_form
+  end
+
+  def check
+    @mentor_form = mentor_form
+    render :new unless @mentor_form.valid?
+  rescue TeachingRecord::RestClient::HttpError
+    render :no_results
+  end
+
+  def create
+    mentor_form.save!
+
+    redirect_to placements_support_school_mentors_path(@school), flash: { success: t(".mentor_added") }
+  end
+
   private
 
+  def mentor_params
+    params.require(:mentor_form)
+          .permit(:first_name, :last_name, :trn)
+          .merge(default_params)
+  end
+
+  def default_params
+    { service: current_service, school: @school }
```

Comment:
Not a blocker. Given we are in the placements service, perhaps we can use `:placements` instead of `current_service`